assets/javascripts/discourse/components/chat-selection-manager.js

```diff
@@ -0,0 +1,117 @@
+import Component from "@ember/component";
+import discourseComputed from "discourse-common/utils/decorators";
+import { clipboardCopyAsync } from "discourse/lib/utilities";
+import { getOwner } from "discourse-common/lib/get-owner";
+import { action } from "@ember/object";
+import { ajax } from "discourse/lib/ajax";
+import { isTesting } from "discourse-common/config/environment";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { schedule } from "@ember/runloop";
+import { inject as service } from "@ember/service";
+import getURL from "discourse-common/lib/get-url";
+
+export default Component.extend({
```

Comment:
can you try using native class syntax please?

Reply:
Yep will do, just copied from the  live pane.